docs/data/building_permits.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
@speedyturkey I think that this yml frontmatter will be invalid. There should just be one pair of `---`, like this:

```
---
layout: datasource
tablename: building_permits
title: Building Permit Data
---
```